tinyagentos/routes/delegation.py

```
@@ -0,0 +1,321 @@
+                project_id=project_id,
+                metadata={
+                    "kind": "delegation_gate",
+                    "from_agent": from_agent,
+                    "to_agent": to_agent,
+                    "task_id": task_id,
+                    "task_title": task_title,
+                    "note": note,
+                },
+            )
+            decision_id = decision["id"]
+        except Exception:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)